Fixes:
C:/users/maarten/.conan/data/ncurses/6.2/_/_/build/970e773c5651dc2560f86200a4ea56c23f568ff9/source_subfolder/ncurses/tinfo/lib_acs.c(48): error C2375: '_nc_acs_map': redefinition; different linkage

(required for shared MSVC)

--- include/curses.h.in
+++ include/curses.h.in
@@ -298,7 +298,7 @@
 /* line graphics */
 
 #if @BROKEN_LINKER@ || NCURSES_REENTRANT
-NCURSES_WRAPPED_VAR(chtype*, acs_map);
+NCURSES_WRAPPED_VAR(NCURSES_IMPEXP chtype*, acs_map);
 #define acs_map NCURSES_PUBLIC_VAR(acs_map())
 #else
 extern NCURSES_EXPORT_VAR(chtype) acs_map[];
